app/lib/bk/compat/parsers/bitrise/github_last_commit_date.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module BitriseSteps
+      # Handles the translation of github-last-commit-date
+      class Translator
+        def translate_github_last_commit_date(inputs)
+          raise 'Invalid github-last-commit-date step configuration!' unless inputs.key?('repository_url')
```

Comment:
I couldn't find this option in [bitrise's plugin documentation](https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/git-last-commit-date/1.0.0/step.yml)